### ISSUE TITLE
Applies to: and content dated on Microsoft Access/Office Custom Ribbon docs

### ISSUE THREAD

**Issue by barnam3** [MicrosoftDocs/office-developer-client-docs#216]
[https://docs.microsoft.com/en-us/office/client-developer/access/desktop-database-reference/how-to-apply-a-custom-ribbon-when-starting-access](url)

Some important content in this doc: is depreciated; last updated 10/15/2018 and states Applies to: Access 2013 | Office 2013.

The Microsoft Office Button hasn't been around for some time; I certainly don't remember seeing it in 2013, which this doc "Applies to:". The data types should be Short Text and Long Text, not Text and Memo. The ID field is also required, but not referenced.

All other support relating to this topic points to this doc, and those list Access/Office 2007-2016 and 365 in the Applies to:.

The top to search results for this topic are this doc and the one at support.office.com, which the latter we can't edit here, but the content is roughly the same, the inclusion of the ID field being the first obvious exception.

To quick start the bare bones roll your own Ribbon users, the first 1/3 of this doc should be enough to make a rudimentary ribbon that runs some macros, but it's not. All 3 docs in this series need updating. [https://docs.microsoft.com/en-us/office/client-developer/access/desktop-database-reference/ribbon](url)

[https://support.office.com/en-us/article/create-a-custom-ribbon-in-access-45e110b9-531c-46ed-ab3a-4e25bc9413de](url)

**Comment by lindalu-MSFT** [comment]
@barnam3 Hi, yes you're right, some of the content for Access is deprecated. We recommend you go to Stack Overflow and ask Access developers [questions about customizing the ribbon](https://stackoverflow.com/search?q=customize+access+ribbon). When I searched, I found several answers there. If you have recommendations how to correct these topics, or what I can do to make them more useful, please let me know. We don't have the resources currently to update them and since this is GitHub, after all, we rely on collaboration from Office developers!

**Comment by barnam3** [comment]
Why was this issue closed? Yes, I do have some suggestions and even though write access is limited, forks can still be created - which I plan to do. I'm not in need of assistance in using the Ribbon, but thanks, I just opened an issue, well, because I assumed that was the proper thing to do.

**Comment by lindalu-MSFT** [comment]
Hi @barnam3 reopening while you work on your suggestions! Please do create a fork, and open a PR. I can then review your suggestions. Once merged, I'll close this issue! I'll be around over the holidays. I'm using this time to work from home and catch up on stale work, such as this! Sorry that we don't have resources to monitor this repo. I do it in my spare time because there's literally no one else. Cheers, and thanks so much for caring enough to help out!

**Comment by barnam3** [comment]
Fair enough - I took my last final today, so I should have time over the Holiday as well. I will probably compose my fork sans depreciation, then we can append/revise it from there if needed. Some of this stuff is so old, I can't imagine anyone would miss it... especially as accessible as Office is these days. Anytime - thanks again!
